.cirrus.yml

```diff
@@ -268,17 +268,22 @@ task:
         - git clone https://github.com/flutter/tests.git bin/cache/pkg/tests
         - dart --enable-asserts dev/customer_testing/run_tests.dart --skip-on-fetch-failure --skip-template bin/cache/pkg/tests/registry/*.test
 
-    - name: firebase_test_lab_tests-linux # linux-only
-      environment:
+    # firebase_test_lab_tests, linux-only
+    - environment:
         # Empirically, this shard runs in 20-25 minutes with just one CPU and 4G of RAM, as of
         # October 2019. It does not seem to be sensitive to the number of CPUs or amount of RAM;
         # doubling CPUs had no effect (mere seconds under 20 minutes), increasing RAM to 24G left it
         # on the high end of the 20-25 minute range. (This makes sense, as it's just driving the
         # Firebase test lab remotely.) Less than 4G of RAM made it go OOM.
         CLOUDSDK_CORE_DISABLE_PROMPTS: 1
         GCLOUD_FIREBASE_TESTLAB_KEY: ENCRYPTED[1c140257edc48f5578fa5a0e5038b84c8e53270c405efa5a8e35ea303a4e0d135853989f448f72136206de854d17fbec]
-      script:
-        - ./dev/bots/firebase_testlab.sh
+      matrix:
+       - name: firebase_lab_release_smoke_test-linux
+         script: ./dev/bots/firebase_testlab.sh release_smoke_test
+       - name: firebase_lab_abstract_method_smoke_test-linux
+         script: ./dev/bots/firebase_testlab.sh abstract_method_smoke_test
+       - name: firebase_lab_android_embedding_v2_smoke_test-linux
+         script: ./dev/bots/firebase_testlab.sh android_embedding_v2_smoke_test
```

Comment:
please don't do it this way -- just use the same style the rest of the file does
we're trying to keep this file to as rigorously simple a style as we can

Reply:
Are you referring to adding a `SHARD_TEMPLATE` for the environment or pushing the shard numbers down into the test?

Reply:
i mean using `matrix`